aiohttp_jwt/middleware.py

```diff
@@ -23,6 +23,7 @@ def JWTMiddleware(
     is_revoked=None,
     store_token=False,
     algorithms=None,
+    header_prefix='Bearer',
```

Comment:
Should be `auth_scheme` or something similar. `header_prefix` is not an obvious naming as for me.